src/api/config/types.ts

```diff
@@ -7,6 +7,10 @@ export interface Config {
   useImageProxy: boolean,
   specialLinks: SpecialLinks,
   version: BackendVersion,
+  plantuml: {
+    enable: boolean,
```

Comment:
I don't think, that we need an extra "enabled"-flag. If the url is set then the plugin is enabled.

Reply:
Ignore this one. We can refactor it, when we extract the code into a plugin.